package.json

```diff
@@ -26,7 +26,7 @@
 		"lint:ts": "eslint . --ext .ts,.tsx",
 		"lint:yaml": "yamllint github-for-jira.sd.yml",
 		"pretest": "NODE_ENV=test run-p clean setup db:test",
-		"test": "jest --runInBand --forceExit --coverage",
+		"test": "jest --runInBand --forceExit --coverage --detectOpenHandles",
```

Comment:
This will give a breakdown of why jest doesn't close cleanly.  We can use it as a todo list of things that we need to fix as part of the tests.